src/OpenTelemetry.Instrumentation.AspNetCore/README.md

```diff
@@ -10,17 +10,15 @@ collect metrics and traces about incoming web requests. This instrumentation
 also collects traces from incoming gRPC requests using
 [Grpc.AspNetCore](https://www.nuget.org/packages/Grpc.AspNetCore).
 
-**Note: This component is based on the OpenTelemetry semantic conventions for
-[metrics](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/http-metrics.md)
+**Note: This component is based on the
+[v1.23](https://github.com/open-telemetry/semantic-conventions/tree/v1.23.0) of
```

Comment:
please add which attributes are added by default, and hint that more attributes may be added by using enrich